src/main/resources/hudson/matrix/Messages_da.properties

```diff
@@ -23,7 +23,7 @@
 TextArea.DisplayName=Brugerdefineret akse
 MatrixProject.DuplicateAxisName=Dupliker akse navn
 MatrixBuild.Triggering=Starter {0}
-LabelAxis.DisplayName=Slaver
+LabelAxis.DisplayName=Agenter
```

Comment:
Do you speak Danish? :grin:

Reply:
I have a good friend of mine who does, Google Translate. Maybe do you know him as well? ;)

Reply:
LOL @javeleon thinking of you